test/conftest.py

```diff
@@ -5,6 +5,7 @@
 from flask import url_for
 from flask_login import current_user
 
+import tracker
```

Comment:
can we just import what we really need? like advisory?

Reply:
Sure, I do this out of habit..